share/functions/__fish_status_to_signal.fish

```diff
@@ -0,0 +1,22 @@
+function __fish_status_to_signal --description "Print signal name from argument (\$status), or just argument"
+    if test (count $argv) -ne 1
```

Comment:
I prefer `if set -q argv[1]` - "the first element of argv is set".

Reply:
That is not exactly the same though as it will accept multiple arguments. To be exactly the same it has to be: `if set -q $argv[1] || not set -q $argv[2]`.   (Oddly the `or` operator does not work here, only the `||`).
I think it is clearer as it is (using `count`) but if you prefer your suggestion and happy to accept multiple arguments then I will change it.

Reply:
I misused the `or` operator by forgetting to put a comma in front of it.

Reply:
I mean semicolon!

Reply:
If `argv[1]` isn't set, `argv[2]` can't either. Variables don't have holes.

Reply:
Exactly, the test as I have it using `count` allows 1 and exactly 1 argument to the function whereas what you are proposing allows 1 or more. To get the exact same behavior as what I have but not using `count` but `set -q` instead would require to verify the presence of `argv[1]` but absence of `argv[2]`.
So dropping `count` but keeping the same behavior would need `if set -q $argv[1] || not set -q $argv[2]` for emitting the error message under the same circumstances.
However saying all that I don't really mind. 
As long as you are happy that your proposal is not exactly the same and you are OK with allowing multiple arguments which are just ignored then I will change it.